bundles/org.openhab.binding.dolbycp/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dolbycp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="cp750">
+		<label>CP750 Device</label>
+		<description>Binding for Dolby Cinema Processor CP750</description>
+
+		<channels>
+			<channel id="fader" typeId="fader"/>
+			<channel id="mute" typeId="mute"/>
+			<channel id="input" typeId="input"/>
+			<channel id="analog" typeId="input-analog"/>
+			<channel id="dig1" typeId="input-dig1"/>
+			<channel id="dig2" typeId="input-dig2"/>
+			<channel id="dig3" typeId="input-dig3"/>
+			<channel id="dig4" typeId="input-dig4"/>
+			<channel id="nonsync" typeId="input-non_sync"/>
+			<channel id="mic" typeId="input-mic"/>
+			<channel id="osversion" typeId="os-version"/>
+		</channels>
+
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the device</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535">
+				<label>Port</label>
+				<default>61408</default>
+				<description>Port to access the device</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="0">
```

Comment:
I thought that a delay of 0 might be an option if the Sound Processor will only be controlled by OpenHab. But I think that won't ever happen (since it has physical buttons and normally is used in conjunction with a theater management system), so I will change this.